src/pages/new/index.tsx

```diff
@@ -18,17 +25,25 @@ const WhatsNewPage: FunctionComponent<any> = ({resource}) => {
     thirdSecondaryResource,
   ] = secondary.resources
 
+  const {background, lightBackground} = jumbotron
+  const {lightFeatureCardBackground, featureCardBackground} = secondPrimary
+
+  const themedBackground = theme === 'light' ? lightBackground : background
+  const themedFeatureCardBackground =
+    theme === 'light' ? lightFeatureCardBackground : featureCardBackground
```

Comment:
check to see what theme is being used and render the proper background.

I haven't handled the case where data is not present. This was starting to get verbose so I thought I'd PR this and get the teams thoughts. This would be why I'd want to extract into a hook